### ISSUE TITLE
Screen reader compatibility mac OS X 10.9 & iOS 7.1

### ISSUE THREAD

**Issue by doyletim** [FortAwesome/Font-Awesome#3273]
My screen reader doesn't read out the icons using Safari on Mac OS X 10.9 and iOS 7.1

**Comment by sensibleworld** [comment]
Doing a little management in Github and (batch) closing out potentially stale issues. If the above still needs attention, please let us know and we’ll re-open!
